Source/Fuse.Reactive.Expressions/LookUp.uno

```diff
@@ -0,0 +1,213 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+
+namespace Fuse.Reactive
+{
+	/** Represents a reactive object-index look-up operation, with a computed index.
+		
+		Index can be either a number (for IArray lookups) or a string (for IObject lookups).
```

Comment:
Could you document what happens with out-of-bounds accesses? And maybe what happens when a fractional number is passed?